tests/integration/tasks/test_genesys_task.py

```diff
@@ -1,65 +1,79 @@
-from viadot.tasks import GenesysToCSV, GenesysToDF
-
-import os
+import pytest
 from unittest import mock
+from datetime import datetime, timedelta
 
-import pandas as pd
-import pytest
+from viadot.tasks import GenesysToCSV
+
+
+@pytest.fixture
+def var_dictionary():
```

Comment:
Maybe add some docstring here. To give a general overview of what it is.

Reply:
done!